platforms/access-bridge/package.json

```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "main": "build/main.js",
   "engines": {
-    "node": ">=18.13.0"
+    "node": ">=21.1.0"
```

Comment:
Sentry version that supports Express middleware, requires higher version of node. Otherwise we cannot use the latest features Sentry offers.